indexer/src/main/java/org/uniprot/store/indexer/literature/reader/LiteratureStatisticsReader.java

```diff
@@ -11,6 +11,11 @@
 public class LiteratureStatisticsReader
         implements RowMapper<LiteratureStatisticsReader.LiteratureCount> {
 
+    public static final String LITERATURE_STATISTICS_SQL =
```

Comment:
Can you please move this constant to LiteratureStatisticsStep class, so you can make it private.